src/models/image-map.test.ts

```diff
@@ -21,7 +21,8 @@ function unsafeUpdate(func: () => void) {
 }
 
 describe("ImageMap", () => {
-  const kCurriculumBaseUrl = "https://example.com/clue-curriculum/";
+  const kCurriculumUnitBaseUrl = "https://example.com/clue-curriculum/branch/main/sas/";
+  const kCurriculumUnitUrl = `${kCurriculumUnitBaseUrl}config.json`;
```

Comment:
A nit, but my inclination would be to not include the `/` in the base url because the downstream urls seem more natural that way:
```suggestion
  const kCurriculumUnitBaseUrl = "https://example.com/clue-curriculum/branch/main/sas";
  const kCurriculumUnitUrl = `${kCurriculumUnitBaseUrl}/config.json`;
```